src/commands/applications/index.ts

```diff
@@ -0,0 +1,61 @@
+import select from '@inquirer/select';
+import { Flags } from '@oclif/core';
+import {
+  deleteApplication,
+  selectApplication,
+} from '../../applications/management';
+import { BaseCommand } from '../../base';
+
+export default class Applications extends BaseCommand {
+  public static description =
+    'List Applications. Requires `application:read` Management Application permission';
+
+  public static examples = ['<%= config.bin %> <%= command.id %>'];
+
+  public static flags = {
+    page: Flags.integer({
+      char: 'p',
+      description: 'applications list page to fetch',
```

Comment:
```suggestion
      description: 'Applications list page to fetch',
```

should this be capital like the others?